packages/frontend-shared/package.json

```diff
@@ -24,48 +24,64 @@
     "@antfu/utils": "^0.3.0",
     "@cypress/vite-dev-server": "0.0.0-development",
     "@cypress/vue": "0.0.0-development",
+    "@graphql-typed-document-node/core": "^3.1.0",
     "@headlessui/vue": "1.4.0",
     "@iconify/json": "1.1.368",
     "@iconify/vue": "3.0.0-beta.1",
     "@intlify/vite-plugin-vue-i18n": "2.4.0",
     "@percy/cypress": "^3.1.0",
     "@testing-library/cypress": "8.0.0",
+    "@types/faker": "5.5.8",
     "@urql/core": "2.3.1",
     "@urql/exchange-execute": "1.1.0",
     "@urql/exchange-graphcache": "4.3.3",
     "@urql/vue": "0.4.3",
-    "@vitejs/plugin-vue": "1.2.4",
-    "@vitejs/plugin-vue-jsx": "1.1.6",
+    "@vitejs/plugin-vue": "2.2.4",
+    "@vitejs/plugin-vue-jsx": "1.3.8",
     "@vue/compiler-core": "3.2.31",
     "@vue/compiler-dom": "3.2.31",
     "@vue/compiler-sfc": "3.2.31",
     "@vueuse/core": "7.2.2",
     "@windicss/plugin-interaction-variants": "1.0.0",
+    "ansi-to-html": "0.6.14",
```

Comment:
A lot of new packages added here - some of them removed from the launchpad package.json, others added here (and still there). Is this just general cleanup, making sure things are required in the right place?

Reply:
A lot of these are legitimate, but there's a few that leaked in while debugging a frontend-shared-only failure